hack/check-rec-rule-usage.sh

```diff
@@ -1,22 +1,41 @@
 #!/usr/bin/env bash
 set -e
-
 TMP=$(mktemp -d)
 echo "Created temporary directory at $TMP"
 
+CONOSLE_DIR=$1
+
+if [ ! -d $CONOSLE_DIR ]; then
+    echo "Directory $CONOSLE_DIR does not exist"
+    exit 1
+fi
+
 findrules="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)/find-rules.sh"
```

Comment:
I notice that `find-rules.sh` processes files that match `*prometheus-rule.yaml` but it misses at least `assets/prometheus-k8s/prometheus-rule-thanos-sidecar.yaml`. It would be better to parse all assets and match on `kind: PrometheusRule`.

Reply:
`find-rules.sh` modified, now using `kind: PrometheusRule` to find out rules.